tests.py

```diff
@@ -1,7 +1,7 @@
 import urllib.request
 import unittest
 from threading import Thread
-
+from unittest.mock import patch
```

Comment:
please keep the extra line in between the python standard library imports and the project imports